legacy_app.py

```
@@ -34,6 +34,17 @@
 )
 from settings import get_runtime_env_name
 
+from app.application_metadata import build_application_metadata
+from app.bootstrap.openapi import (
```

Flake8 (7.3.0)

[error] 'app.bootstrap.openapi._OPENAPI_ALLOWED_EXACT' imported but unused

(F401)